src/pages/Proxies.tsx

```diff
@@ -192,7 +207,12 @@ export default () => {
                     <div class="space-y-2">
                       <div class="flex items-center justify-between pr-8">
                         <div class="flex items-center">
-                          <Show when={proxyGroup.icon}>
+                          <Show
+                            when={
+                              proxyGroup.icon &&
```

Comment:
Also take advantage of javascript feature [Optional Chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)

Reply:
Done